modular_skyrat/modules/assault_operatives/code/interrogator.dm

```diff
@@ -0,0 +1,218 @@
+#define STAGE_PROCESS_TIME_LOWER 30 SECONDS
+#define STAGE_PROCESS_TIME_UPPER 1 MINUTES
+#define ALERT_CREW_TIME 25 SECONDS
+
+/**
+ * The interrorgator, a piece of machinery used in assault ops to extract GoldenEye keys from heads of staff.
+ *
+ * This device has 3 stages.
+ *
+ * This device has a few requirements to function:
+ * 1. Must be on station Z-level
+ * 2. Must be a head of staff with a linked interrogate objective
+ * 3. Must be alive
+ * 4. Must not be a duplicate key
+ *
+ * After a key has been extracted, it will send a pod somewhere into maintenance, and the syndicates will know about it straight away.
+ */
+
+/obj/machinery/interrogator
```

Comment:
It's useful documentation, let it be AUTODOC'd.
```suggestion
/**
 * The interrorgator, a piece of machinery used in assault ops to extract GoldenEye keys from heads of staff.
 *
 * This device has 3 stages.
 *
 * This device has a few requirements to function:
 * 1. Must be on station Z-level
 * 2. Must be a head of staff with a linked interrogate objective
 * 3. Must be alive
 * 4. Must not be a duplicate key
 *
 * After a key has been extracted, it will send a pod somewhere into maintenance, and the syndicates will know about it straight away.
 */
/obj/machinery/interrogator
```